### PR TITLE
Enable nullability in MenuCommandService

### PR DESCRIPTION
**Pull request by halgab**
## Proposed changes

- Enable nullability in `MenuCommandService`

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/9992)

### CODE REVIEW

File: src/System.Windows.Forms.Design/src/System/ComponentModel/Design/MenuCommandService.cs [hunk 1]
```diff
@@ -294,23 +288,24 @@ protected void EnsureVerbs()
...
-                    verbsOrder.Add(localVerbs[i]);
-                    buildVerbs[key] = verbsOrder.Count - 1;
+                    DesignerVerb localVerb = localVerbs[i]!;
+                    verbsOrder.Add(localVerb);
+                    buildVerbs[localVerb.Text] = verbsOrder.Count - 1;
                 }
             }
 
             // look for duplicate, prepare the result table
+            //TODO: isn't 'result' too big if there are duplicates?
```
Comment (halgab) [inline src/System.Windows.Forms.Design/src/System/ComponentModel/Design/MenuCommandService.cs:308]: I'm worried we're missing an `Array.Resize(ref result, j);` after the loop below...

Comment (dreddy-work) [reply src/System.Windows.Forms.Design/src/System/ComponentModel/Design/MenuCommandService.cs:308]: Based on logic, `buildVerbs ` and `result` both will have same size.  `verbsOrder` size may be bigger (dues to duplicates).

**Review by dreddy-work — DISMISSED** [review]
LGTM

File: src/System.Windows.Forms.Design/src/System/ComponentModel/Design/MenuCommandService.cs (hunk 1 above)
Comment (halgab) [reply src/System.Windows.Forms.Design/src/System/ComponentModel/Design/MenuCommandService.cs]: Okay, I get it now. Thanks! I removed the useless comment

**Review by dreddy-work — APPROVED** [review]

**Pull request merged**